OpenStack-Rabbit-Consumer/etc/openstack-utils/rabbit-consumer/aq_manage.py

```diff
@@ -17,20 +18,21 @@ def fix_json(fake_json):
     """Used after getting json from openstack which returns a list of dictionaries with field, value pairs"""
     if isinstance(fake_json, str): fake_json = json.loads(fake_json)
     out = {}
-    for i in fake_json: out[i["Field"]] = i["Value"]
+    for i in fake_json:
+        out[i["Field"]] = i["Value"]
     return out
 
 def attempt_url(url, retry=5):
     """Will attempt a aquilon url command a default of 5 times. Prints instead of returning"""
     auth = HTTPKerberosAuth()
     out = req_ses.post(url, auth=auth).text
-    #kerberos authentication goes here
-    #if out != "No data" and out != "":
-        #subprocess.call(["kinit","-k"])
-        #auth = HTTPKerberosAuth()
-    print(out)
+    #kerberos authentication
+    if out != "No data\n" or out != "":
+        subprocess.call(["kinit","-k"])
```

Comment:
This needs to be able to include a string after the -k i.e.
`kinit -k HTTP/server`
This should come from the config file